src/test/java/com/fasterxml/jackson/databind/deser/creators/SingletonDelegatingCreatorTest.java

```diff
@@ -0,0 +1,78 @@
+package com.fasterxml.jackson.databind.deser.creators;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+
+import static com.fasterxml.jackson.databind.testutil.DatabindTestUtil.newJsonMapper;
+import static org.junit.jupiter.api.Assertions.assertSame;
+
+// [databind#4688]
+public class SingletonDelegatingCreatorTest
+{
+
+    static final class NoFieldSingletonWithDelegatingCreator {
+        private static final NoFieldSingletonWithDelegatingCreator INSTANCE = new NoFieldSingletonWithDelegatingCreator();
+
+        private NoFieldSingletonWithDelegatingCreator() {}
+
+        @JsonCreator(mode = JsonCreator.Mode.DELEGATING)
```

Comment:
Ok so would the idea here be to basically drop any JSON value that would match this type?
(I guess it must).

This was not something intended to work, although if it did use to work there's something to be said for not changing behavior. Will add a note on issue itself.